quirks/password-rules.json

```diff
@@ -222,6 +234,12 @@
     "spirit.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [!@#$%^&*()];"
     },
+    "splunk.com": {
+        "minlength: 8; maxlength: 64; required: lower; required: upper; required: digit; required: [!@#$%&*_-+=<>];"
+    },
+    "sulamericaseguros.com.br": {
+        "password-rules": "minlength: 6; maxlength: 6;"
```

Comment:
And all ascii-printable characters are allowed? Or are there any restrictions?

Reply:
all ascii-printable allowed.